ddtrace/profiling/exporter/http.py

```diff
@@ -78,7 +79,7 @@ def __attrs_post_init__(self):
         tags = {
             k: six.ensure_str(v, "utf-8")
             for k, v in itertools.chain(
-                parse_tags_str(os.environ.get("DD_TAGS")).items(),
+                gitmetadata.update_tags(parse_tags_str(os.environ.get("DD_TAGS"))).items(),
```

Comment:
Could you add comments why we are doing that?
I think I understand why, but I'm not entirely sure and that can be quite confusing for the reader who doesn't have a lot of context.